src/main/java/httpServer/ListenerAndResponder.java

```diff
@@ -51,9 +52,8 @@ public void readClientInput(Socket clientSocket) throws IOException {
         if (!httpRequest.equals("")) {
             HTTPRequest request = RequestParser.parse(httpRequest);
             System.out.println(httpRequest);
-            StatusCodes statusCodes = router.getResponse(request);
-            String responseText = statusCodes.httpResponse;
-            printServerResponse(responseText, printer);
+            String httpResponse = router.getResponse(request);
+            printServerResponse(httpResponse, printer);
```

Comment:
Consider moving line 44, where you create the printer, into the `printServerResponse` method.

Reply:
Done!